content/en/docs/Reference/operator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Operators"
+linkTitle: "Operators"
+weight: 3
+date: 2021-03-15
+---
+<br>
```

Comment:
You need to remove this inline html as it's breaking the markdown rendering of the next header.